localstack-core/localstack/services/apigateway/next_gen/execute_api/handlers/integration_request.py

```diff
@@ -77,13 +79,20 @@ def __call__(
         if not (integration_method := integration["httpMethod"]) or integration_method == "ANY":
             # otherwise, fallback to the request's method
             integration_method = context.invocation_request["method"]
+        # AWS doesn't send a body for these methods. Mock needs the body to get the status code
+        # TODO this might need to be done in the integrations?
```

Comment:
I agree with the comment, I think it can/should be up to the integration to use the body or not 👍especially since those have strong control about how they create their request, and we can see `MOCK` receives a body somehow.

Reply:
It could also lead to a bit of code duplication if they all need to check for the HTTP request before sending however. If we see that's the case, maybe we could migrate it back here? We can also leave it as is for now, I'm not too opinionated about this!

Reply:
I will remove that and the tests with it then!